index.html

```diff
@@ -134,6 +151,60 @@ <h5>SEE THE WHOLE PROGRAM</h5>
 				<a class="join-button" href=""> Join LOTR Celebration 2022</a>
 			</section>
 		</main>
+		<section id="featured-actors" class="featured-actors">
+			<h2>Featured Characters</h2>
+			<hr />
```

Comment:
- [ ] [_OPTIONAL_] It's better to use two columns for the Featured Characters part and also better to include the background checker images for the images of the characters.

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/169389587-815d8ca7-0386-4f2d-b89a-e6aaa00d0e4e.png)

### Your Implementation 
![image](https://user-images.githubusercontent.com/59416136/169389702-685ad54e-89b8-4e7d-b2d4-c87e9cdd4532.png)

Reply:
hey @Wes-Isaac ! Thank you for the review! that idea about the columns was great, it's more beautiful now. Hope now is all finished!
Hi @ShadyShawkat ! I was confused whit the footer because the different versions and pages have different styles. So I upload it in the wrong way. Thank you for noticing that!